0.16/files/docker-entrypoint.sh

```diff
@@ -50,13 +50,34 @@ if ! find -L "$SAVES" -iname \*.zip -mindepth 1 -print | grep -q .; then
     --map-settings "$CONFIG/map-settings.json"
 fi
 
+#-z string True if the string is null
```

Comment:
I would prefer if you did not update 0.16 or 0.17. I didn't touch the code in a longer while and don't want to break it. Also the comments are not required and this script is still sh not bash.